api/sql/schema/750-rule.decision.lookup.sql

```diff
@@ -172,6 +172,44 @@ BEGIN
         --destination category
         ('dc', 'destination.account.product', @destinationAccountProductId)
 
+    IF OBJECT_ID(N'customer.customer') IS NOT NULL
+    BEGIN
+        IF @sourceOwnerId IS NOT NULL
+            INSERT INTO
+                @operationProperties(factor, name, value)
+            SELECT
+                'sk', 'source.kyc', c.kycId
+            FROM
+                customer.customer c
+            WHERE
+                c.actorId = @sourceOwnerId
+            UNION SELECT
+                'st', 'source.customerType', ct.customerTypeNumber
+            FROM
+                customer.customer c
+            LEFT JOIN
+                customer.customerType ct ON ct.customerTypeId = c.customerTypeId
+            WHERE
+                c.actorId = @sourceOwnerId
+        IF @destinationOwnerId IS NOT NULL
+            INSERT INTO
+                @operationProperties(factor, name, value)
+            SELECT
+                'dk', 'destination.kyc', c.kycId
+            FROM
+                customer.customer c
+            WHERE
+                c.actorId = @destinationOwnerId
+            UNION SELECT
+                'dt', 'destination.customerType', ct.customerTypeNumber
+            FROM
+                customer.customer c
+            LEFT JOIN
+                customer.customerType ct ON ct.customerTypeId = c.customerTypeId
+            WHERE
+                c.actorId = @destinationOwnerId
+    END
+
```

Comment:
Match the customer type / customer kyc only when the customer module is present.